ui/components/multichain/network-list-menu/add-non-evm-account/add-non-evm-account.tsx

```diff
@@ -21,19 +23,100 @@ import {
   TextAlign,
 } from '../../../../helpers/constants/design-system';
 import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { getMetaMaskHdKeyrings } from '../../../../selectors';
+import { CreateSnapAccount } from '../../create-snap-account';
+import { SrpList } from '../../multi-srp/srp-list';
+import { WalletClientType } from '../../../../hooks/accounts/useMultichainWalletSnapClient';
+
+const SNAP_CLIENT_CONFIG_MAP: Record<
+  string,
+  { clientType: WalletClientType | null; chainId: CaipChainId | null }
+> = {
+  [MultichainNetworks.BITCOIN]: {
+    clientType: WalletClientType.Bitcoin,
+    chainId: MultichainNetworks.BITCOIN,
+  },
+  [MultichainNetworks.BITCOIN_TESTNET]: {
+    clientType: WalletClientType.Bitcoin,
+    chainId: MultichainNetworks.BITCOIN_TESTNET,
+  },
+  [MultichainNetworks.SOLANA]: {
+    clientType: WalletClientType.Solana,
+    chainId: MultichainNetworks.SOLANA,
+  },
+};
```

Comment:
I've seen this before somewhere

Reply:
It's a similar map, but is using network instead of action (used in account menu) as a key